docs/android-core/stage-management/1-introduction.mdx

```diff
@@ -50,73 +48,62 @@ meeting.stage.join()
+
+- StageDisabledForMeetingType: The meeting type does not support stage management.
+- PermissionDenied: The user does not have the required permissions to perform the operation.
+- ActionInvalidForStageStatus: The operation is invalid for the current stage status.
+- NoRequestToCancel: There is no pending request to cancel.
+
 ### List of Stage Events
 
-The `DyteStageEventListener` interface provides callback methods for various stage events. Implement these callbacks to handle 
+The `DyteStageEventListener` interface provides callback methods for various stage events. Implement these callbacks to handle
 stage-related events in your application:
 
 ```kotlin
 meeting.addStageEventsListener(object : DyteStageEventListener {
-    override fun onPresentRequestReceived() {
-        // Called when the local user's stage access request is accepted by the host,
-        // or when the local user, who is a viewer, is invited to the stage by the host.
+    override fun onStageAccessRequestAccepted() {
+        // Called when the local user is accepted to join the stage.
     }
 
-    override fun onAddedToStage() {
-        // Called when the local user successfully joins the stage.
+    override fun onStageAccessRequestRejected() {
+        // Called when the local user's request to join the stage is rejected by the host.
     }
 
     override fun onRemovedFromStage() {
         // Called when the local user is removed from the stage.
     }
 
-    override fun onPresentRequestAdded(participant: DyteJoinedMeetingParticipant) {
-        // Called when a participant requests to join the stage. Triggered only if the local user is a host.
-    }
-
-    override fun onPresentRequestClosed(participant: DyteJoinedMeetingParticipant) {
-        // Called when a participant with a pending stage access request leaves the meeting.
-        // Triggered only if the local user is a host.
-    }
-
-    override fun onPresentRequestRejected(participant: DyteJoinedMeetingParticipant) {
-        // Called when a participant's stage access request is denied by the host.
-        // Triggered only if the local user is a host.
-    }
-
-    override fun onPresentRequestWithdrawn(participant: DyteJoinedMeetingParticipant) {
-        // Called when a participant cancels their stage access request.
-        // Triggered only if the local user is a host.
-    }
-
-    override fun onParticipantRemovedFromStage(participant: DyteJoinedMeetingParticipant) {
-        // Called when a participant is removed from the stage by the host.
+    override fun onNewStageAccessRequest(participant: DyteRemoteParticipant) {
+        // Called when a new stage access request is received from a participant.
     }
 
-    override fun onStageRequestsUpdated(accessRequests: List<DyteJoinedMeetingParticipant>) {
+    override fun onStageAccessRequestsUpdated(accessRequests: List<DyteRemoteParticipant>) {
         // Called when the list of stage access requests is updated.
```

Comment:
(change-request): Can we add the following part about host in the comment?
"Triggered only if the local user is a host."